controllers/gardens.js

```diff
@@ -0,0 +1,70 @@
+const mongoose = require("../db/connection.js")
+const db = mongoose.connection
+const router = require("express").Router()
+const Garden = require("../models/garden.js")
+const Plant = require("../models/Plant.js")
```

Comment:
Not sure whether or not it matters since you said things are working, but your Model name is "plant" not "Plant"

Reply:
Nice catch, I'm guessing it's figuring it out what the right file is anyway but I changed it to be consistent.